COLLADAMaya/include/COLLADAMayaPhysicSceneExporter.h

```diff
@@ -0,0 +1,82 @@
+/*
+    Copyright (c) 2008-2009 NetAllied Systems GmbH
```

Comment:
Please change copyright of new files, this is not a NetAllied file, its a Starbreeze file created in 2015